packages/nartus_geocoder/pubspec.yaml

```diff
@@ -0,0 +1,55 @@
+name: nartus_geocoder
+description: A new Flutter package project.
+version: 0.0.1
+homepage:
+
+environment:
+  sdk: ">=2.19.1 <3.0.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+  geocoding: ^2.0.5
+dev_dependencies:
```

Comment:
add an empty line between `dependencies` and `dev_dependencies` group. It just look nicer